functions/__fzf_preview_file.fish

```diff
@@ -28,6 +29,6 @@ function __fzf_preview_file --argument-names file_path --description "Print a pr
     else if test -p "$file_path"
         __fzf_report_file_type "$file_path" "named pipe"
     else
-        echo "File doesn't exist or is empty." >&2
+        echo "$file_path doesn't exist." >&2
```

Comment:
Outputting file_path because it helps with debugging.
Removing the "is empty" part because empty files count as regular files and so will result in an empty preview.